releasenotes/notes/remove-quantum-instance-and-runtime-2c00393c0de1b9a9.yaml

```diff
@@ -0,0 +1,7 @@
+---
+upgrade:
+  - |
+    Support for ``QuantumInstance``-based algorithms has been removed.
+  - |
+    Classes ``VQEClient`` and ``QAOAClient`` have been removed.
```

Comment:
Perhaps we should say that using primitives under a Session with the Runtime is the "replacement" function for these.